patches/net/minecraft/server/players/PlayerList.java.patch

```diff
@@ -48,11 +48,38 @@
          ServerLevel serverlevel = p_11287_.serverLevel();
          p_11287_.awardStat(Stats.LEAVE_GAME);
          this.save(p_11287_);
+@@ -453,7 +_,14 @@
+         } else {
+             optional = Optional.empty();
+         }
+-
```

Comment:
Don't remove the initial whitespace between the prior closing brace and the patch